### ISSUE TITLE
Minimizing popout windows places them in their own bar above the taskbar

### ISSUE THREAD

**Issue by Felanbird** [Chatterino/chatterino2#4180]
Likely caused by #3836

### Discussed in https://github.com/Chatterino/chatterino2/discussions/4179

<div type='discussions-op-text'>

<sup>Originally posted by **Hotair10** November 21, 2022</sup>
In previous versions of Chatterino, when minimizing a popup chat, it would simply disappear into the taskbar. This paired perfectly with my "Always on Top" setting I had enabled, as it would be out of the way when I didn't want it blocking my screen, such as playing a game on that monitor.
In the newest update, it now minimizes into a little tab above my taskbar like this:
![image](https://user-images.githubusercontent.com/5515786/203149145-bf69c5a9-5420-4bc9-b8e7-afaccea534c4.png)

With the "Always on Top" setting enabled, this little tab is also always on top. While the tab can be moved around freely, it'd be nice if there was an option to disable the feature entirely and have it operate as it did in previous versions.

As a side note, it continues to add additional blips for each popup chat you minimize. No information on channel or anything, just a repeated list of the application title, making the feature even less useful than it could be.
![image](https://user-images.githubusercontent.com/5515786/203149669-6d7344b2-e14a-44f2-9746-b139f5f24b06.png)

tl;dr: Add a setting for old minimize behavior to still be an option.</div>
